common/crypto/CMakeLists.txt

```diff
@@ -39,23 +39,23 @@ ENDIF()
 # by the client (ias verification requires sgx).
 ################################################################################
 IF (BUILD_TRUSTED OR BUILD_UNTRUSTED)
-  SET(PROJECT_GENERATED_IAS_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/verify_ias_report/ias-certificates.cpp)
-  SET_SOURCE_FILES_PROPERTIES(${PROJECT_GENERATED_IAS_SOURCES} PROPERTIES GENERATED TRUE)
-
-  ADD_CUSTOM_COMMAND(
-    OUTPUT  ${PROJECT_GENERATED_IAS_SOURCES}
-    COMMAND ./build_ias_certificates_cpp.sh
-    DEPENDS verify_ias_report/ias-certificates.template verify_ias_report/build_ias_certificates_cpp.sh
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/verify_ias_report
+  SET(CRYPTO_BUILD_DIR ${CMAKE_BINARY_DIR}/crypto)
+  SET(INTEL_ROOT_CACERT_PATH ${CRYPTO_BUILD_DIR}/Intel_SGX_Attestation_RootCA.pem)
+  SET(IAS_CERTIFICATES_H_PATH ${CRYPTO_BUILD_DIR}/ias-certificates.h)
+
+  FILE(DOWNLOAD
+      https://certificates.trustedservices.intel.com/Intel_SGX_Attestation_RootCA.pem
+      ${INTEL_ROOT_CACERT_PATH}
+      TLS_VERIFY ON
+  )
+  FILE(READ
+      ${INTEL_ROOT_CACERT_PATH}
+      INTEL_ROOT_CA_PEM
+  )
+  CONFIGURE_FILE(
+      ${CMAKE_CURRENT_SOURCE_DIR}/verify_ias_report/ias-certificates.h.in
+      ${IAS_CERTIFICATES_H_PATH}
```

Comment:
i suggest you add a custom command to the clean target to remove these

Reply:
one more... it might also be useful to turn on "show progress" for this file

Reply:
> i suggest you add a custom command to the clean target to remove these

We could, but we would end up in a situation where cmake generates the artifacts and make cleans them up.
In that case, it's probably better to keep the script for fetching/generating these artifacts, which is called by make.

Keeping these artifacts in the build folder may still make sense considering that:
* the certificate is valid until 2049 and
* the header is rebuilt (by cmake!) when the source changes.

Reply:
> one more... it might also be useful to turn on "show progress" for this file

done